README.md

```diff
@@ -1 +1,3 @@
-# genesis
\ No newline at end of file
+# Genesis
+
+This repository houses the open-source components of Mattermost Genesis. This tool is providing a service to provision AWS Accounts and manage VPC CIDR pools, VPC infrastructure and VPC peering functionality.
```

Comment:
I can imagine that you have already in your list to document the details.

Reply:
Yeap the readme is added in another PR ;)